README.md

```diff
@@ -58,19 +58,19 @@ You can configure the following environment variables for deploying your stack u
 
 Few hints, to customize the [`docker-compose.yml`](docker-compose.yml) orchestration according to your needs:
 
-* To use data persistence (which we recommend in production settings), the *Mongo* service needs to be deployed in **global mode**. This is to avoid that more than one instance is deployed on the same node and that different instances concurrently access the same MongoDB data space on the filesystem.
+* To use data persistence (which we recommend in production settings), the *mongo* service needs to be deployed in **global mode** (see [`docker-compose.yml` line 5](docker-compose.yml#L5)). This is to avoid that more than one instance is deployed on the same node and that different instances concurrently access the same MongoDB data space on the filesystem.
```

Comment:
IMO lines numbers are quite fragile here, they'll become outdated quite soon and will be more a confusion than help.

Reply:
of course these has to be updated in case. let's see.